test/test_meta.py

```diff
@@ -331,7 +332,6 @@
     torch.nn.functional.adaptive_avg_pool2d,  # MISSING aten::_adaptive_avg_pool2d
     torch.nn.functional.adaptive_avg_pool3d,  # MISSING aten::_adaptive_avg_pool3d
     torch.nn.functional.batch_norm,  # MISSING aten::native_batch_norm
-    torch.nn.functional.binary_cross_entropy,  # MISSING aten::binary_cross_entropy
```

Comment:
Are the `# missing` parts autogenerated btw?

Reply:
yup